appraisal.lic

```diff
@@ -13,21 +13,39 @@ class Appraisal
     settings = get_settings
     train_list = settings.appraisal_training
 
-    train_list.each do |task|
-      break if DRSkill.getxp('Appraisal') >= 30
-      case task
-      when 'zills'
-        assess_zills
-      when 'pouches'
-        train_appraisal_with_pouches(settings.full_pouch_container, settings.gem_pouch_adjective)
-      when 'gear'
-        train_appraisal_with_gear
-      when 'bundle'
-        train_appraisal_with_bundle
-      when 'art'
-        train_appraisal_with_art
-      else
-        echo "#{task} is not a valid appraisal_training setting"
+    arg_definitions = [
+      [
```

Comment:
I _think_ that this will allow `;appr focus` or `;appr broadsword` since both args are optional.

I _think_ this will prevent this problem:

```
arg_definitions = [
      [
	        { name: 'focus', regex: /focus/i, description: 'Perform appraise focus on an item.' },
	        { name: 'item', regex: /\w+/i, description: 'Item to use appraise focus with.' }
       ],
       []
]
```

❓ Could you test these two cases?